cwms_radar_api/src/main/java/cwms/radar/api/ClobController.java

```diff
@@ -89,9 +92,7 @@ protected DSLContext getDslContext(Context ctx)
                                            @OpenApiContent( type = Formats.JSONV2, from = Clobs.class ),
                                            @OpenApiContent( type = Formats.XMLV2, from = Clobs.class )
                                        }
-                      ),
-                      @OpenApiResponse(status="501",description = "The format requested is not implemented"),
```

Comment:
Since we are still returning non-200 status codes, should they still be listed here?
HttpServletResponse.SC_NOT_IMPLEMENTED

Reply:
If you look in RadarAPI.java you'll notice that the codes+format returned by everything are listed in the default confiugration. I just happened to miss this instance.

Reply:
In theory we should be able to do the same for the office, but I didn't want to work that into this PR.